testutil/singlevm/verify.sh

```diff
@@ -126,6 +126,33 @@ else
 	echo "Container connectivity verified"
 fi
 
+#Clear out all prior events
+"$ciao_gobin"/ciao-cli event delete
+
+#Wait for the event count to drop to 0
+retry=0
+ciao_events=0
+
+until [ $retry -ge 6 ]
+do
+	ciao_events=`"$ciao_gobin"/ciao-cli event list | grep "0 Ciao event" | wc -l`
```

Comment:
@markdryan will add another PR to actually peek into the event and verify the data.